src/main/java/com/internship/socialnetwork/model/User.java

```diff
@@ -16,27 +17,32 @@
 @Data
 @NoArgsConstructor
 @Entity
+@Table(name = "app_user")
```

Comment:
Why didi you changed name of table? Can we set it back to user you can use "`user`"